deploy/scripts/src/testIPVCore.ts

```diff
@@ -529,11 +531,15 @@ export function coreScenario1 (): void {
       )
       const endTime1 = Date.now()
 
-      check(res, {
+      passed = check(res, {
```

Comment:
We can remove it from this check as well. Let us keep it only on the last check of the last request